src/providers/personalData/store.tsx

```diff
@@ -107,13 +107,17 @@ export const usePersonalDataStore = createStore<PersonalDataStoreState, Personal
     persist: {
       key: 'personalData',
       whitelist: WHITELIST,
-      version: 0,
+      version: 1,
       onRehydrateStorage: () => {
         WHITELIST.forEach((item) => {
           window.localStorage.removeItem(item)
         })
       },
-      migrate: () => null,
+      migrate: (oldState) => ({
+        recentSearches: oldState.recentSearches.filter(
+          (item: RecentSearch) => !Object.prototype.hasOwnProperty.call(item, 'type')
+        ),
+      }),
```

Comment:
This removed all the other data except `recentSearches` from `personalData` store